stage_environment.yml

```diff
@@ -0,0 +1,22 @@
+# An environment file for Project Officers wanted to use Conda rather than
+# virtualenv
+# This is based on scripts/setup_virtualenv.sh and doesn't interfere with the
+# wheel package
+# conda env create --file=stage_environment.yml
+# conda env update --file=stage_environment.yml
+name: python-aodndata-conda
+variables:
+  STAGE: production
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - cython
+  - python=3.6
```

Comment:
Why not 3.8, to match what we'll have in production next?

Reply:
I was having some compilation issues. I'll try again and put the issues below

Reply:
@mhidas 
with python 3.8, i get the following error:
```
The conflict is caused by:
    aodntools 1.4.7 depends on pandas<0.25.0 and >=0.24.2
    aodntools 1.4.3 depends on pandas<0.25.0 and >=0.24.2
    aodntools 1.4.1 depends on pandas<0.25.0 and >=0.24.2
    aodntools 1.3.3 depends on pandas<0.25.0 and >=0.24.2
    aodntools 1.3.2 depends on pandas<0.25.0 and >=0.24.2

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict
```

Reply:
I wonder if this would fix it? https://github.com/aodn/python-aodntools/pull/142